dist/components/tabs/_tabs.dust

```diff
@@ -0,0 +1,20 @@
+{?tabs}
+    <div class="c-tabs" role="tab">
+        <ul id="c-tab-controls" class="c-tabs__controls" role="tablist">
```

Comment:
I think this is something we want to solve generically for components going forward. It may not be very likely for tabs, but it could happen. And other types of UI that have similar requirements for ARIA or for wiring JS could easily have multiple instances on the page. Let’s loop in an engineer or two to help figure that out.